vaapi/vaapidisplay.cpp

```diff
@@ -128,6 +128,8 @@ class NativeDisplayDrm : public NativeDisplayBase{
     };
```

Comment:
Please ignore this patch, I keep it here just make sure the decode api unit test runnable. I have other pull request to address this problem. https://github.com/01org/libyami/pull/649. I will rebase this after it merged